test/e2e/utils_test.go

```diff
@@ -227,7 +227,7 @@ func getCertManagerOperatorSubscription(ctx context.Context, loader library.Dyna
 // patchSubscriptionWithCloudCredential uses the k8s dynamic client to patche the only Subscription object
 // in the cert-manager-operator namespace to inject CLOUD_CREDENTIALS_SECRET_NAME="aws-creds" env
 // into its spec.config.env
-func patchSubscriptionWithCloudCredential(ctx context.Context, loader library.DynamicResourceLoader) error {
+func patchSubscriptionWithCloudCredential(ctx context.Context, loader library.DynamicResourceLoader, credentialValue string) error {
```

Comment:
@geliu2016 Because we are contributing upstream, even if for minor thing like variable name, it is better to be precise. Here, it is not a credential "value", it is just a secret "name", so the variable `credentialValue` is not much precise. How about "credentialSecret"?
Same for other places `credentialValue := "aws-creds"`

Reply:
updated.